### ISSUE TITLE
Symfony 6.3 deprecations

### ISSUE THREAD

**Issue by laurentmuller** [createnl/zxcvbn-bundle#13]
I just updated my project to Symfony 6.3.0. A deprecation is now raised:

`Method "Symfony\Component\DependencyInjection\Extension\ExtensionInterface::load()" might add "void" as a native return type declaration in the future. Do the same in implementation "Createnl\ZxcvbnBundle\DependencyInjection\ZxcvbnBundleExtension" now to avoid errors or add an explicit @return annotation to suppress this message.`

The deprecation is in the following `ZxcvbnBundleExtension` class. It is just matter to update return type:

Before:
```php
public function load(array $configs, ContainerBuilder $container)
```

After:
```php
public function load(array $configs, ContainerBuilder $container): void
```

Or:
```php
/**
  * @return void
  */ 
public function load(array $configs, ContainerBuilder $container)
```

**Comment by AlexLisenkov** [comment]
Fixed in version 3.0.1

**Comment by laurentmuller** [comment]
Thank You.
